examples_tests/22.RaytracedAO/raytraceCommon.glsl

```diff
@@ -42,7 +48,9 @@ layout(set = 2, binding = 4) restrict coherent buffer RayCount // maybe remove c
 layout(set = 2, binding = 5, r32ui) restrict uniform uimage2DArray albedoAOV;
 layout(set = 2, binding = 6, r32ui) restrict uniform uimage2DArray normalAOV;
 // environment emitter
-layout(set = 2, binding = 7) uniform sampler2D envMap; 
+layout(set = 2, binding = 7) uniform sampler2D envMap;
+layout(set = 2, binding = 8) uniform sampler2D warpMap; 
+layout(set = 2, binding = 9) uniform sampler2D luminance[MAX_LUMINANCE_LEVELS];
```

Comment:
could you use an actual ImageView with a full mip-map chain?

Would be a lot cleaner code.

Reply:
Done